TCMSApp/src/client/app/layout/ht-top-nav.directive.js

```diff
@@ -34,6 +54,13 @@
                 return $state.current.title.substr(0, menuName.length) === menuName ? 'active' : '';
             }
 
+            function logOut() {
+                var Trello = authservice.authorize();
```

Comment:
code looks strange. to logOut we need to authorize first? Actually we need to be already authorized.

Trello is our constant that is injected to the authservice. It is much better to leave Trello object only there.

so that this functioni should be only 2 rows:
```js
authservice.deauthorize(); 
$state.go('index');```